### PR TITLE
Test the intl-memoizer concurrent mode

### PR DESCRIPTION
**Pull request by gregtatum**
This is part of my effort to add to the test coverage for the project.

### CODE REVIEW

File: intl-memoizer/src/lib.rs [hunk 1]
```diff
@@ -407,4 +408,46 @@ mod tests {
...
+    }
+
+    #[test]
+    fn test_debug_single_thread() {
+        let memoizer = IntlLangMemoizer::new("en".parse().unwrap());
+        assert!(
+            format!("{:?}", memoizer).starts_with("IntlLangMemoizer"),
+            "The memoizer debug prints."
+        );
+    }
```
Comment (nordzilla) [inline intl-memoizer/src/lib.rs:452]: > thought

I'm a bit unclear on what this is testing that is of use. Any object that derives `Debug` will print _something_ that contains the object's type name.

**Review by nordzilla — APPROVED** [review]
This looks good to me, though I don't understand the purpose of the debug tests.

File: intl-memoizer/src/lib.rs (hunk 1 above)
Comment (gregtatum) [reply intl-memoizer/src/lib.rs:452]: I was dogmatically testing anything that was uncovered, which the Debug trait was being listed as uncovered. Maybe it's not the best use of a test.

Comment (nordzilla) [reply intl-memoizer/src/lib.rs:452]: Hmm. My preference would be if there was a way to ignore `Debug` trait from the coverage report. I don't know if we have that ability, but I would like that more than having tests that don't really test anything substantial just for the sake of having coverage here. It's not the end of the world to keep them, but I don't think we gain anything from this other than a higher number on the report.

Comment (gregtatum) [reply intl-memoizer/src/lib.rs]: Weirdly enough, the coverage isn't even showing up as missing there now.

**Pull request merged**